src/client_input_handler.rs

```diff
@@ -570,27 +618,56 @@ fn make_move(am_i_player_one: bool, cur_game_state: &GameState) -> Msg {
         stdin
             .read_line(&mut move_to_make)
             .expect("Error reading in");
-        let move_to_make_int: usize;
-        match move_to_make.trim().parse() {
-            Ok(x) => {
-                move_to_make_int = x;
-            }
-            Err(e) => {
-                println!("invalid integer + {}!", e);
-                continue;
-            }
+
+        if !verify_move_to_make(move_to_make.trim()) {
+            println!("Invalid move entered!");
+            move_to_make = String::new();
+            continue;
         }
-        if check_is_move_valid(move_to_make_int, cur_game_state, am_i_player_one) {
-            final_move = move_to_make_int;
+
+        if check_is_move_valid(
+            move_to_make.parse::<usize>().unwrap(),
```

Comment:
I think this is causing a panic:

```Current game state:
           #13: 4 | #12: 4 | #11: 4 | #10: 4 |  #9: 4 | #8: 4
        0 --------+--------+--------+--------+--------+-------- 0
            #1: 4 |  #2: 4 |  #3: 4 |  #4: 4 |  #5: 4 | #6: 4
Player 1, enter your move (1 - 6)
asdvjnsavd
could not make move into an int: invalid digit found in string!
Invalid move entered!
Player 1, enter your move (1 - 6)
99999999999
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ParseIntError { kind: InvalidDigit }', src\libcore\result.rs:1188:5```

When trying to unwrap a digit that is too big